### ISSUE TITLE
Compatability with Exchange Server

### ISSUE THREAD

**Issue by phixitvt** [evrenvarol/outlook-taskboard#25]
I am having issues with getting the taskboard to populate with any tasks at work. I use it at home and love it. At home I have Outlook 2016 running standalone, but at work we are on Outlook 2010 using an exchange server. I was wondering if anyone has experience using taskboard on 2010 or an exchange server. Thanks!

**Comment by gautambastian** [comment]
I'm having the same problem, but with Outlook 2016 running on an Exchange Server. The taskboard is not getting populated. When I use [+] on the taskboard to add a Task, it does get created but it doesn't appear in the Backlog, which remains unpopulated along with the rest of the Tasklanes.

**Comment by phixitvt** [comment]
Yep, same behavior as I am seeing. Good to know that it probably isn't an Outlook 2010 vs. 2016 thing.

**Comment by BillyMcSkintos** [comment]
Outlook 2010 and Exchange at the office for me and not having problems, although it only works in IE for me (I display the dashboard in the browser).

@gautambastian Have you checked that 1. the config paths of your tasks are correct and 2. You created the folders under the root task folder?

**Comment by tazer808** [comment]
I am seeing the same behavior on Outlook 2007 / 2013 (Taskboard not populating), connected to work MS Exchange server. 

I can create new tasks using the "+" button and they populate in the proper folders (same as @phixitvt and @gautambastian) but do not display in the HTML homepage set for the Task Board. 

@BillyMcSkintos All folders have been created under the root task folder, and I have checked the folder paths in config.js to make sure all columns are looking for the created folders but even the 'default' Backlog column doesn't show up which as I understand it is by default looking at the root "Tasks" path. As a side note, even the IE version in-browser does not populate.
